core/libp2p/crypto/common.hpp

```diff
@@ -38,14 +36,6 @@ namespace libp2p::crypto::common {
    */
   using Aes256Secret = AesSecret<32, 16>;
 
-  /**
-   * Public and private keys
-   */
-  struct KeyPair {
```

Comment:
Please, remove structure from common and move it to separate file in the same PR.